container/raw/handler.go

```diff
@@ -200,6 +200,16 @@ func (self *rawContainerHandler) GetSpec() (info.ContainerSpec, error) {
 		if utils.FileExists(cpuRoot) {
 			spec.HasCpu = true
 			spec.Cpu.Limit = readUInt64(cpuRoot, "cpu.shares")
```

Comment:
@jimmidyson  hm not really , why not but i think this should be done in a different PR